src/admins/admins.controller.spec.ts

```diff
@@ -17,4 +24,52 @@ describe('AdminsController', () => {
   it('should be defined', () => {
     expect(controller).toBeDefined();
   });
+
+  // https://fakerjs.dev/guide/
+  it('should create new admin', async () => {
+    const newAdmin = {
+      name: faker.name.fullName(),
+      email: faker.internet.email(),
+      phone: faker.random.numeric(11),
+      username: String(faker.internet.userName()),
+      password: faker.internet.password(),
+      role: 'ADMIN',
+      designation: faker.name.jobArea(),
+    } as CreateAdminDto;
+    const response = await controller.create(newAdmin);
+    expect(response.status).toEqual('success');
+    expect(response.data).toBeDefined();
+    adminId = response.data['id'];
+    adminInfo = newAdmin;
+  });
+
+  it('should find all admins', async () => {
+    const response = await controller.findAll();
+    expect(response.status).toEqual('success');
+    expect(response.data).toBeDefined();
+  });
+
+  it("should find the recently created admin's information", async () => {
+    const response = await controller.findOne(String(adminId));
+    expect(response.status).toEqual('success');
+    expect(response.data).toBeDefined();
+    expect(response.data['name']).toEqual(adminInfo.name);
+  });
```

Comment:
Make separate methods for different test cases.

Reply:
If I am not wrong each of these are different methods already. And this is the popular process for writing test cases of a single class. We could use 'Describe()' tag. But async was not working properly for that tag.

Reply:
Definitely all test cases should be in a single class, but you can put different test cases in different methods. Like the following image.

![Screenshot 2022-08-22 at 11 16 03 AM](https://user-images.githubusercontent.com/41617780/185844329-0935121b-9247-4ab4-8425-52b5ad7f3b2a.png)

Reply:
As I have cleared my point, can I Merge the code?